Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM ubuntu:14.04
 
+EXPOSE 8050 8051
```

Comment:
Cool. Personally, I think the best way to organize instruction in Dockerfile, is placing fast instruction at the end of Dockerfile. Then, if we change expose for a particular case, we can reuse cache.

Also, fast instruction like EXPOSE don't really need cache.

Remember, cache can't be reuse if we want to change the first instruction.